### PR TITLE
{bio}[GO] gofasta v0.0.5 created

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/gofasta/gofasta-0.0.5.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+source_urls = ['https://github.com/cov-ert/gofasta/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['2d66f1bfd68cc673b4991634eedda14faf5f8c3e6ca578b18d68b3763a45606b']
+
+builddependencies = [
+    ('Go', '1.16.5'),
+]
+
+# options = {'modulename': 'absl'}
```
Comment (Micket) [inline easybuild/easyconfigs/g/gofasta/gofasta-0.0.5.eb:24]: ?

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/gofasta/gofasta-0.0.5.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/g/gofasta/gofasta-0.0.5.eb:24]: I really need new glasses. Thanks for spotting this.

File: easybuild/easyconfigs/g/gofasta/gofasta-0.0.5.eb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/cov-ert/gofasta/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['2d66f1bfd68cc673b4991634eedda14faf5f8c3e6ca578b18d68b3763a45606b']
+
+builddependencies = [
+    ('Go', '1.16.5'),
+]
+
```
Comment (Micket) [inline easybuild/easyconfigs/g/gofasta/gofasta-0.0.5.eb:23]: Please add `sanity_check_commands = ['gofasta -h']`

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/gofasta/gofasta-0.0.5.eb (hunk 2 above)
Comment (Micket) [reply easybuild/easyconfigs/g/gofasta/gofasta-0.0.5.eb:23]: Maybe also sanity_check_files, or maybe gopackage adds something useful there by default? I'm not familiar with this easyblock

Comment (sassy-crick) [reply easybuild/easyconfigs/g/gofasta/gofasta-0.0.5.eb:23]: Good idea. I have added this and done the sanity check for the file and folder as well.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/53382982556380257c8ad209221455f4 for a full test report.

**Review by Micket — APPROVED** [review]
kgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Pull request merged**